apps/src/templates/AchievementDialog.jsx

```diff
@@ -195,26 +205,69 @@ const AchievementDialog = Radium(React.createClass({
         handleClose={this.handleClose.bind(this, !tooManyBlocks)}
         assetUrl={this.props.assetUrl}
       >
-        <div style={styles.checkmarks}>
-          {this.achievementRow(true, locale.puzzleCompleted())}
-          {this.achievementRow(!tooManyBlocks, this.blocksUsedMessage(blockDelta, params))}
-          {this.achievementRow(!tooManyHints, this.hintsMessage(tooManyHints))}
-        </div>
-        {this.props.showStageProgress &&
-          <div style={styles.stageRewards}>
-            <div style={styles.stageRewardsTitle}>
-              {locale.stageRewards()}
-            </div>
-            <div style={styles.progressBackground}>
-              <div
-                style={{
-                  ...styles.progressForeground,
-                  width: this.props.stageProgress * MAX_PROGRESS_WIDTH
-                }}
-              />
-            </div>
-          </div>
-        }
+        <StaggeredMotion
+          defaultStyles={Array(4).fill({ progress: 0 })}
```

Comment:
Pretty sure babel-polyfill is taking care of it for us. Either way, I tested on saucelabs and it works fine in IE11.